types/n3/index.d.ts

```diff
@@ -152,17 +169,17 @@ export interface WriterConstructor {
 export const Writer: WriterConstructor;
 
 export interface N3Writer {
```

Comment:
The changes to the input args are not needed I think, only those to the return types.